### PR TITLE
remote remote connector from LB4

### PR DESCRIPTION
**Pull request by dhmlau**
Since remote connector is not supported in LB4, we're removing it from the LB4 docs. 

Another PR will be submitted in loopback-next on the sidebar.

### CODE REVIEW

File: update-readmes.sh [hunk 1]
```diff
@@ -28,7 +28,6 @@ strongloop loopback-connector-openapi master
 strongloop loopback-connector-oracle master
 strongloop loopback-oracle-installer master
 strongloop loopback-connector-postgresql master
-strongloop loopback-connector-remote master
```
Comment (agnes512) [inline update-readmes.sh]: Will this affect LB3 docs https://loopback.io/doc/en/lb3/Remote-connector.html ?

**Comment by dhmlau** [comment]
cc @bajtos

File: update-readmes.sh (hunk 1 above)
Comment (dhmlau) [reply update-readmes.sh]: Taking a closer look at the script, it actually will affect LB3 readme too. Let me remove it. Thanks.

**Review by agnes512 — APPROVED** [review]

**Review by bajtos — APPROVED** [review]
LGTM, thanks!

**Pull request merged**